exporters/variables-scss/src/generators/cssObjectGenerator.ts

```diff
@@ -1,51 +1,121 @@
-import { Token, TokenGroup } from '@supernovaio/sdk-exporters';
-import { toPlural } from '../helpers/stringHelper';
+import { Token, TokenGroup, TokenType } from '@supernovaio/sdk-exporters';
 import { tokenVariableName } from '../helpers/tokenHelper';
+import { toPlural } from '../helpers/stringHelper';
 
-export const generateObjectContent = (
-  tokens: Array<Token>,
+export const COLOR_SUFFIX = '-colors';
+
+export type CssObjectType = { [key: string]: string | object };
+
+/* This function handles cases that are outside the logic of aliases for the remaining tokens.
+A common condition is that for tokens with a numeric part, the non-numeric part is dropped.
+Non-numeric tokens are left in their original form. Deviations from this logic are addressed here.
+e.g. radius-full -> full */
+const invariantTokenAlias: { [key: string]: string } = {
+  'radius-full': 'full',
+};
+
+export const normalizeFirstNamePart = (part: string, tokenType: TokenType): string => {
+  if (tokenType === TokenType.color) {
+    return `$${part}${COLOR_SUFFIX}`;
+  }
+
+  return `$${toPlural(part.toLowerCase())}`;
+};
+
+export const handleInvariantTokenAlias = (tokenName: string): string => {
+  if (invariantTokenAlias[tokenName]) {
```

Comment:
I think two returns are too much for this ;-) In my opinion, a ternary operator would be better in this case.